terraform/admin_vm.tf

```diff
@@ -0,0 +1,42 @@
+locals {
+  create_fip = var.env_name != "production" && var.zabbix_web_access ? true : false
```

Comment:
Is there a use case for _not production and no zabbix web access_?  If no, let's remove `zabbix_web_access var`